src/Controller/UsersController.php

```diff
@@ -476,6 +507,20 @@ public function getUserMapper(PDO $db, Request $request)
         return $this->userMapper;
     }
 
+    public function setTalkCommentMapper(TalkCommentMapper $talkCommentMapper): void
+    {
+        $this->talkCommentMapper = $talkCommentMapper;
+    }
+
+    public function getTalkCommentMapper(PDO $db, Request $request): TalkCommentMapper
```

Comment:
Since the return value of this method is currently never used, perhaps it would make more sense to rename this method to `initializeTalkCommentMapper()`, and add a `void` return type declaration?